### PR TITLE
Feature files for user stories

### PR DESCRIPTION
**Pull request by wallidhgs**
* Adding feature files for user stories in https://github.com/wallidhgs/poc_bdd/pull/2
* Removing login feature file from template

### CODE REVIEW

File: features/account/accesibility.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Feature: User settings page
+    As a logged user, in any page
+    I want to toggle Use High Contrast UI from The left menu in the account section
+    So I can view page more easily
```
Comment (cesarrecio-enroute) [inline features/account/accesibility.feature]: pages

Comment (cesarrecio-enroute) [reply features/account/accesibility.feature]: @ORIGIN-Ticket-008
Feature: Accesibility
As a logged user, on any page, I want to be able to toggle Use High Contrast UI  from the left menu in the Account section. So the visibility for the user is way better.

File: features/account/accesibility.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Ticket 008
+
+    Scenario: Activate high contrast ui
```
Comment (cesarrecio-enroute) [inline features/account/accesibility.feature]: Scenario: The user should be able to Activate high-contrast UI.

File: features/account/accesibility.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        Given the user is logged in
+        And the user is on "dashboard" page
+        And the user open the "account" side menu
+        When the user click toggle "true" the "high contrast" check
+        And the page is refreshed
+        Then the page should be displayed in high contrast
+
+    Scenario: Deactivate high contrast ui
```
Comment (cesarrecio-enroute) [inline features/account/accesibility.feature]: Scenario: The user should be able to Deactivate high-contrast UI.

File: features/account/settings.feature [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Feature: User settings page
```
Comment (cesarrecio-enroute) [inline features/account/settings.feature]: @ORIGIN-Ticket-007
Feature: User Settings Page
As a logged user, on the settings page, I want to be able to see the user profile details. The user should be able to update the information.

File: features/account/settings.feature [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    As a logged user, in my settings page
+    I want to see my profile details
+    So I can change my information
+    Ticket 007
+
+    Scenario: Display user settings
```
Comment (cesarrecio-enroute) [inline features/account/settings.feature:6]: Te falta `Scenario Outline`

File: features/course/homepage.feature [hunk 6]
```diff
@@ -0,0 +1,26 @@
+Feature: Course homepage
+    As an logged instructor, in the course home page
+    I want see the link to the section pages
+    So I can easily navigate and edit the course
+    Ticket 005
+
+    Scenario: Display edit course sections
```
Comment (cesarrecio-enroute) [inline features/course/homepage.feature:7]: ditto, you need `Scenario outline`

File: features/course/people/add_people.feature [hunk 7]
```diff
@@ -0,0 +1,26 @@
+Feature: Course people page
```
Comment (cesarrecio-enroute) [inline features/course/people/add_people.feature]: @ORIGIN-Ticket-009
Feature: Add People Page
As a logged instructor, on the course people page, I want to be able to add students or teachers by email. All the students and teachers should receive the proper invitation to the course.

File: features/course/people/add_people.feature [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    As a logged instructor, in the course people page
+    I want to add students or teachers in a batch by email
+    So they can have an invitation to the course
+    Ticket 009
+
+    Scenario: Add student
```
Comment (cesarrecio-enroute) [inline features/course/people/add_people.feature]: Scenario: The instructor should be able to add a student

File: features/course/people/add_people.feature [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+        Given the user is logged in
+        And the user is on "course people page" page
+        When the user click on "people" button
+        And fill the add people form with "students"
+        Then the page should display the "students" as pending
+
+    Scenario: Add teachers
```
Comment (cesarrecio-enroute) [inline features/course/people/add_people.feature]: Scenario: The instructor should be able to add a teachers

File: features/course/people/add_people.feature [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+        Given the user is logged in
+        And the user is on "course people page" page
+        When the user click on "people" button
+        And fill the add people form with "teachers"
+        Then the page should display the "teachers" as pending
+
+    Scenario: Add teacher assistants
```
Comment (cesarrecio-enroute) [inline features/course/people/add_people.feature]: Scenario: The instructor should be able to add a teacher assistants

File: features/course/people/remove_people.feature [hunk 11]
```diff
@@ -0,0 +1,11 @@
+Feature: Course people page
```
Comment (cesarrecio-enroute) [inline features/course/people/remove_people.feature]: @ORIGIN-Ticket-009
Feature: Remove People Page
As a logged instructor, on the course people page, I want to be able to remove students or teachers.

**Review by cesarrecio-enroute — CHANGES_REQUESTED** [review]
a couple of comments

File: features/account/accesibility.feature [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+    So I can view page more easily
+    Ticket 008
+
+    Scenario: Activate high contrast ui
+        Given the user is logged in
+        And the user is on "dashboard" page
+        And the user open the "account" side menu
+        When the user click toggle "true" the "high contrast" check
+        And the page is refreshed
+        Then the page should be displayed in high contrast
```
Comment (Cesar-AGG) [inline features/account/accesibility.feature]:
```
Given the option of changing the contrast is available through the "account" side menu
When the "high contrast" option is set to "true"
Then the website is displayed in high contrast
```
The when on this sentence can be different but I don't know the implementation, another option could be the following if it makes more sense based on the current implementation
```
When the "high contrast" option is enabled
```
This way, we have a very flexible Gherkin where technically every option in whatever section in the side menu could be manipulated and validated, only the `Then` would be different

**Pull request merged**